src/performance/perfcollect/perfcollect

```diff
@@ -1463,7 +1540,6 @@ DoCollect()
 			sleep 1
 		done
 	fi
-	EndCollect
```

Comment:
Same here.  More sleep functionality.  Can you make sure that the diff is just your intended changes?  I'm sure that what happened is that you copied a file into your repo after these updates were made, and so they show up as deletes.